examples/find_gamma_eff.py

```diff
@@ -0,0 +1,14 @@
+from pylab import *
+import sys
+from pyspecdata import *
+import SpinCore_pp
+
+config_dict = SpinCore_pp.configuration('active.ini')
+print("Your original gamma effective was:", config_dict['gamma_eff_MHz_G'])
+Delta_nu = float(sys.argv[1])/1e6
+new_gamma = config_dict['gamma_eff_MHz_G'] * (1 - (Delta_nu/config_dict['carrierFreq_MHz'])) 
```

Comment:
now doesn't call Field but uses the carrier frequency and gamma effective. This was tested and produces the same new gamma_effective that I get by hand from the output equation of hahn ech